setup.cfg

```diff
@@ -5,24 +5,32 @@ description-file =
     README.rst
 author = Redfish dev team
 author-email = python-redfish@mondorescue.org
-home-page = http://github.com/bcornec/python-redfish
+home-page = https://github.com/uggla/python-redfish/tree/devel
+license = Apache-2
 classifier =
-    #Environment :: OpenStack
-    Intended Audience :: Information Technology
-    Intended Audience :: System Administrators
-    License :: OSI Approved :: Apache Software License
-    Operating System :: POSIX :: Linux
-    Programming Language :: Python
-    Programming Language :: Python :: 2
-    Programming Language :: Python :: 2.7
-    Programming Language :: Python :: 2.6
-    Programming Language :: Python :: 3
-    Programming Language :: Python :: 3.3
-    Programming Language :: Python :: 3.4
+	Development Status :: 4 - Beta
+		Environment :: Console
+		Intended Audience :: Information Technology
+		Intended Audience :: System Administrators
+		License :: OSI Approved :: Apache Software License
+		Operating System :: POSIX :: Linux
+		Programming Language :: Python
+		Programming Language :: Python :: 2
+		Programming Language :: Python :: 2.7
+		Programming Language :: Python :: 3
+		Programming Language :: Python :: 3.4
 
 [files]
 packages =
-    redfish
+   redfish
+
+scripts =
+	redfish-client/redfish-client.py
+
+data_files =
+	usr/share/redfish-client/templates = redfish-client/templates/*
+	etc/ = redfish-client/etc/*
```

Comment:
Same as above, need to agree on keyword.
To my mind, the best way of doing that will be to put all the substitution into pb conf files.
Then I could probably read those files and do the substitution.
Unfortunately I can not invoke pb using "pip". As I can not create a dep on a rpm/deb or perl pkg.
You I think you just need to rewrite pb in python ! :stuck_out_tongue_winking_eye: :boom: